unifiedWSHandler.js

```diff
@@ -180,167 +173,161 @@ const trackedAlerts = [
  */
-const alertEvent = async function(payload, environment, pgClient, discordClient){
-    if(payload.metagame_event_state_name == "started"){
-        let server = serverNames[payload.world_id];
-        let response = await alertInfo(payload, environment);
-        if(typeof(response.name) != undefined && response.name){
-            let sendEmbed = new MessageEmbed();
-            sendEmbed.setTitle(response.name);
-            if(trackedAlerts.indexOf(Number(payload.metagame_event_id)) == -1){
-                sendEmbed.setDescription(response.description);
+async function alertEvent(payload, environment, pgClient, discordClient){
+    if(payload.metagame_event_state_name !== "started"){
+        return;
+    }
+    const response = await alertInfo(payload, environment);
+    if(response.name === undefined){
+        return;
+    }
+    const sendEmbed = new MessageEmbed();
+    sendEmbed.setTitle(response.name);
+    if(trackedAlerts.indexOf(Number(payload.metagame_event_id)) == -1){
+        sendEmbed.setDescription(response.description);
+    }
+    else{
+        sendEmbed.setDescription(`[${response.description}](https://ps2alerts.com/alert/${payload.world_id}-${payload.instance_id}?utm_source=auraxis-bot&utm_medium=discord&utm_campaign=partners)`);
+    }
+    sendEmbed.setTimestamp();
+    if (response.name.includes('Enlightenment')){
+        sendEmbed.setColor('PURPLE');
+    }
+    else if (response.name.includes('Liberation')){
+        sendEmbed.setColor('BLUE');
+    }
+    else if (response.name.includes('Superiority')){
+        sendEmbed.setColor('RED');
+    }
+    const server = serverNames[payload.world_id];
+    sendEmbed.addField('Server', server, true);
+    sendEmbed.addField('Status', `Started <t:${Math.floor(Date.now()/1000)}:R>`, true);
+    let terObj = {};
+    try{
+        terObj = await territory.territoryInfo(payload.world_id);
+    }
+    catch{
+        
+    }
+    let continent = "Other";
+    let showTerritory = false;
+    if(response.name.toLowerCase().indexOf("indar") > -1){
+        continent = "Indar";
+        showTerritory = true;
+    }
+    else if(response.name.toLowerCase().indexOf("esamir") > -1){
+        continent = "Esamir";
+        showTerritory = true;
+    }
+    else if(response.name.toLowerCase().indexOf("amerish") > -1){
+        continent = "Amerish";
+        showTerritory = true;
+    }
+    else if(response.name.toLowerCase().indexOf("hossin") > -1){
+        continent = "Hossin";
+        showTerritory = true;
+    }
+    else if(response.name.toLowerCase().indexOf("oshur") > -1){
+        continent = "Oshur";
+        showTerritory = true;
+    }
+    else if(response.name.toLowerCase().indexOf("koltyr") > -1){
+        continent = "Koltyr";
+        showTerritory = true;
+    }
+    if(response.name.toLowerCase().indexOf("anomalies") > -1){
+        showTerritory = false;
+    }
+    if(showTerritory && terObj != "" && continent != "Koltyr"){
```

Comment:
Why `{}` on line 205 but `""` here?  I know JS equality checking is a mess but this seems confusing

Reply:
my mistake

Reply:
https://github.com/RemainNA/auraxis-bot/pull/64/commits/634c7815008fd7b0c98ab9527361f43bf3decbe5